install.sh

```diff
@@ -27,27 +27,33 @@ ltsp-config lts.conf
 # Creating client image
 ltsp-update-image --cleanup /
 
+# enabling password authentication 
+sed -i /etc/ssh/sshd_config -e \
+    "/^PasswordAuth/ c PasswordAuthentication yes" 
```

Comment:
- [ ] Move the `-e` switch on the second line.
- [ ]  What if PasswordAuthentication is commented (the line starts with a `#`)?

Reply:
I can use 
`sed -i /etc/ssh/sshd_config -e "/Authentication no\$/ c PasswordAuthentication yes"`
It would handle this case.